ui/store/institutional/institution-background.test.js

```diff
@@ -97,4 +123,52 @@ describe('Institution Actions', () => {
       expect(setWaitForConfirmDeepLinkDialog).toBeDefined();
     });
   });
+
+  describe('#showInteractiveReplacementTokenBanner', () => {
```

Comment:
Can you add test for error handling as well :) 
```
if (err) {
        dispatch(displayWarning(err.message));
        throw new Error(err.message);
      }
```

Reply:
Hey @DDDDDanica, at the moment we are not going to test the error parts